src/python/pants/backend/python/buildozer/buildozer.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import subprocess
+
+from pants.base.build_environment import get_buildroot
+from pants.base.exceptions import TaskError
+from pants.binaries.binary_util import BinaryUtil
+from pants.task.task import Task
+
+
+class Buildozer(Task):
```

Comment:
`src/python/pants/backend/python` pertains to the python language support in pants, so it probably isn't the best place for this code.

how about sticking this into a new `contrib` plugin (e.g. `pantsbuild.pants.contrib.buildozer`)? or worst case a new dir in `src/python/pants/backend/...`.

Reply:
+1 on contrib